app/config/scopes.php

```diff
@@ -31,6 +31,12 @@
     'files.write' => [
         'description' => 'Access to create, update, and delete your project\'s storage files',
     ],
+    'buckets.read' => [
+        'description' => 'Access to read your project\'s storage buckets',
+    ],
+    'buckets.write' => [
+        'description' => 'Access to create, update, and delete your project\'s storage buckets',
```

Comment:
We should use the same format description as other models. We can update this one or all the others.